functions/Export-DbaDbRole.ps1

```diff
@@ -97,40 +97,40 @@ function Export-DbaDbRole {
     .EXAMPLE
         PS C:\> Export-DbaDbRole -SqlInstance sql2005 -Path C:\temp
 
-        Exports the Database Roles for all  for SQL Server "sql2005" and writes them to the file "C:\temp\sql2005-logins.sql"
+        Exports the Database Roles for SQL Server "sql2005" and writes them to the file "C:\temp\sql2005-logins.sql"
 
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -ExcludeLogin realcajun -SqlCredential $scred -Path C:\temp\logins.sql -Append
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -ExcludeRole realcajun -SqlCredential $scred -Path C:\temp\roles.sql -Append
 
-        Authenticates to sqlserver2014a using SQL Authentication. Exports all logins except for realcajun to C:\temp\logins.sql, and appends to the file if it exists. If not, the file will be created.
+        Authenticates to sqlserver2014a using SQL Authentication. Exports all roles except for realcajun to C:\temp\roles.sql, and appends to the file if it exists. If not, the file will be created.
 
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -Login realcajun, netnerds -Path C:\temp\logins.sql
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -Role realcajun,netnerds -Path C:\temp\roles.sql
 
-        Exports ONLY logins netnerds and realcajun FROM sqlserver2014a to the file  C:\temp\logins.sql
+        Exports ONLY roles netnerds and realcajun FROM sqlserver2014a to the file C:\temp\roles.sql
 
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -Login realcajun, netnerds -Database HR, Accounting
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2014a -Role realcajun,netnerds -Database HR, Accounting
 
-        Exports ONLY logins netnerds and realcajun FROM sqlserver2014a with the permissions on databases HR and Accounting
+        Exports ONLY roles netnerds and realcajun FROM sqlserver2014a with the permissions on databases HR and Accounting
 
     .EXAMPLE
         PS C:\> Get-DbaDatabase -SqlInstance sqlserver2014a -Database HR, Accounting | Export-DbaDbRole
 
-        Exports ONLY logins FROM sqlserver2014a with permissions on databases HR and Accounting
+        Exports ONLY roles FROM sqlserver2014a with permissions on databases HR and Accounting
 
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Login realcajun, netnerds -Path C:\temp\login.sql -ExcludeGoBatchSeparator
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql
 
-        Exports ONLY logins netnerds and realcajun FROM sqlserver2008 server, to the C:\temp\login.sql file without the 'GO' batch separator.
+        Exports ONLY roles netnerds and realcajun FROM sqlserver2008 server, to the C:\temp\roles.sql file.
```

Comment:
The way this example was updated it is a duplicate of the example 3. This should be updated to show use of the `-BatchSeparator` parameter.

Reply:
Would you prefer:

```
PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql -BatchSeparator "GO"
```

or

```
PS C:\> Set-DbatoolsConfig -FullName formatting.batchseparator -Value "GO"
PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql
```

The `-BatchSeparator` is defaulted with the following: 
```
...
[string]$BatchSeparator = (Get-DbatoolsConfigValue -FullName 'Formatting.BatchSeparator'),
...
```

Reply:
The default of `formatting.batchseperator` is `GO` already. The description of the example stated `without the 'GO' batch separator`. I don't have test setup to run but that would either be setting `-BatchSeparator $null` or using `Set-DbatoolsConfig` to set the configuration value to `$null`.

Reply:
Will include both! Thanks for the feedback.